src/examples/echo_client.c

```diff
@@ -0,0 +1,74 @@
+/**
+ * Takes a compression encoding and a string from params
+ * and submits them to the echo_server for decompression testing
+ * 
+ * Verifies it gets back the same string as it sent.
+ */
+
+#include <mtev_http.h>
+#include <curl/curl.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+size_t process_response(char *ptr, size_t size, size_t nmemb, void *userdata)
```

Comment:
this will break in case libcurl gets a partial response for part of the string (so that `process_response` gets called multiple times)... which is fine for an example, but I'd like a comment making note of that.